### PR TITLE
Internal: Convert the site settings Gap control for kits to beta [11952]

### PR DESCRIPTION
**Pull request by davseve**

### CODE REVIEW

File: app/modules/import-export/utils.php [hunk 1]
```diff
@@ -115,4 +115,20 @@ public static function get_import_sessions( $should_run_cleanup = false ) {
 
 		return $import_sessions;
 	}
+
+	public static function update_space_between_widgets_values( $space_between_widgets ) {
+		$setting_not_exist = ! isset( $space_between_widgets['size'] );
```
Comment (TzviRabinovitch) [inline app/modules/import-export/utils.php:120]: maybe use positive 
settting_exist 
?

File: core/upgrade/upgrades.php [hunk 2]
```diff
@@ -882,10 +883,8 @@ private static function maybe_add_gap_control_data( $option_name ) {
 			return;
 		}
 
-		$size = strval( $kit_data_array['space_between_widgets']['size'] );
-		$kit_data_array['space_between_widgets']['column'] = $size;
-		$kit_data_array['space_between_widgets']['row'] = $size;
-		$kit_data_array['space_between_widgets']['isLinked'] = true;
+		$kit_data_array['space_between_widgets'] = Utils::update_space_between_widgets_values( $kit_data_array['space_between_widgets'] );
+
```
Comment (TzviRabinovitch) [inline core/upgrade/upgrades.php:887]: do we need this empty line ?

**Review by TzviRabinovitch — CHANGES_REQUESTED** [review]

File: app/modules/import-export/utils.php (hunk 1 above)
Comment (davseve) [reply app/modules/import-export/utils.php:120]: OK, But we will have to negate it in the condition.

Comment (TzviRabinovitch) [reply app/modules/import-export/utils.php:120]: it wont work otherwise :)

Comment (davseve) [reply app/modules/import-export/utils.php]: We can't always be positive :)

**Review by TzviRabinovitch — DISMISSED** [review]

**Review by TzviRabinovitch — DISMISSED** [review]

File: app/modules/import-export/utils.php (hunk 1 above)
Comment (louiswol94) [reply app/modules/import-export/utils.php]: LOL

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**